packages/table/src/Table.ts

```diff
@@ -19,9 +19,9 @@ import {
     SpectrumElement,
     TemplateResult,
 } from '@spectrum-web-components/base';
-import '../sp-table-row.js';
-import '../sp-table-checkbox-cell.js';
-import '../sp-table-body.js';
+import '@spectrum-web-components/table/sp-table-row.js';
+import '@spectrum-web-components/table/sp-table-checkbox-cell.js';
+import '@spectrum-web-components/table/sp-table-body.js';
```

Comment:
The move to bare modules has to do with the tests being able to run through two independent registrations, right?

Reply:
Yep, but it's proved unnecessary, so I can revert these if you like. Just left them for consistency with the rest of the lib.

Reply:
I'd say "consistency" would be the way they were. `@spectrum-web-components/...` is _supposed to be_ for tests, stories, and referencing one package from another. A bit of a fine line between that and everything using it, however. Let's move them back for just now, but maybe take a convo on this pattern to make sure we all know/can benefit from it going forard.